src/store/modules/recipeStore.js

```diff
@@ -1,99 +1,111 @@
+import { data } from "jquery";
+import { createClient } from "pexels";
+
 export const recipeStore = {
-	state: () => ({
-		recipe: {},
-		hasClapped: 0,
-		maxAllowedClaps: 10
-	}),
-	mutations: {
-		INCREMENT_CLAP(state, claps) {
-			this.state.recipe.claps = claps
-			state.hasClapped += 1
-		},
-		UPDATE_RECIPE_STATE(state, newState) {
-			this.state.recipe = newState
-		},
-		RESET_HASCLAPPED(state) {
-			state.hasClapped = 0
-		},
-		COMMENT_POSTED(state, breadcrumb) {
-			location.reload()
-		},
-		COMMENT_DELETED(state) {
-			location.reload()
-		}
-	},
-	actions: {
-		addClap(context, payload) {
-			let url = process.env.BASE_URL + 'add-clap';
+  state: () => ({
+    recipe: {},
+    hasClapped: 0,
+    maxAllowedClaps: 10,
+  }),
+  mutations: {
+    INCREMENT_CLAP(state, claps) {
+      this.state.recipe.claps = claps;
+      state.hasClapped += 1;
+    },
+    UPDATE_RECIPE_STATE(state, newState) {
+      this.state.recipe = newState;
+      state.hasClapped += 1;
+    },
+    RESET_HASCLAPPED(state) {
+      state.hasClapped = 0;
+    },
+  },
+  actions: {
+    addClap(context, payload) {
+      let url = process.env.BASE_URL + "add-clap";
+
+      this.state.api.client
+        .post(
+          url,
+          {
+            recipe_id: payload.recipeId,
+          },
+          this.state.api.options
+        )
+        .then(function (response) {
+          if (response.data.updated) {
+            context.commit("INCREMENT_CLAP", response.data.claps);
+          }
+        })
+        .catch(function (error) {
+          // console.log('clapping error', error.response)
```

Comment:
This was written before the global error handler was implemented. Can you please use the global handler here instead of commenting it out?

```
context.commit('HANDLE_ERROR', error.response)
```
Please test on you local to ensure error.response has the properties needed in the handler